.github/workflows/metabase.yml

```diff
@@ -0,0 +1,23 @@
+name: Azafea Metabase Documentation
+on:
+  push:
+    branches:
+      - master
+
+jobs:
```

Comment:
Aren't we missing the update to RTD?

Reply:
We discussed this verbally – RTD is updated by a web hook.